lib/src/screens/home_screen.dart

```diff
@@ -1,10 +1,71 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
 
-class HomeScreen extends StatelessWidget {
+import 'package:mobile_app/src/screens/login_screen.dart';
+import 'package:mobile_app/src/screens/signup_screen.dart';
```

Comment:
relative import

Reply:
sorry, these 2 imports are added to check bottomnavigator is really working or not, and i forgot to remove it.